itemlist.lua

```diff
@@ -0,0 +1,223 @@
+--[[
+    show_item_list form for Minetest mods
+    (c) Pierre-Yves Rollo
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
```

Comment:
Are you sure? This project is licensed as zlib.
https://github.com/minetest-mods/letters/blob/master/license.txt

Reply:
Sorry, Too quick on copy paste :) I'll relicense.

Reply:
Unwanted licensing removed, code under zlib as for the rest of the mod